src/views/Join/index.js

```diff
@@ -73,10 +73,12 @@ class Join extends Component {
     const party = await this.db.getPartyById(partyId);
 
     if (!party) {
-      // alert(`No party with id ${partyId}`);
+      // eslint-disable-next-line no-alert
+      alert(`No party with id ${partyId}`);
```

Comment:
Rather than using `alert` we can use React Native's `Alert.alert` which is configurable and eslint's no-alert compliant